### PR TITLE
[KotlinCleanUp] KeyboardShortcutIntegrationTest

### PR DESCRIPTION
**Pull request by UvrajSB**
## Purpose / Description
Kotlin cleanup

## Fixes
part of #10489 

## How Has This Been Tested?

Physical device : Realme 8i

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by raymondwzeng — APPROVED** [review]
Seems good to me. Nothing out of the scope of the Kotlin clean-up comments, and all of the requested changes seem to have gone through.

File: AnkiDroid/src/test/java/com/ichi2/anki/reviewer/KeyboardShortcutIntegrationTest.kt [hunk 1]
```diff
@@ -150,19 +149,18 @@ class KeyboardShortcutIntegrationTest : RobolectricTest() {
         assertThat(mReviewer.audioView!!.status, equalTo(recording))
     }
 
-    @KotlinCleanup("scope function")
-    private fun setupPlayerMock(): AudioPlayer {
+    private fun setupPlayerMock() = AudioPlayer().run {
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/anki/reviewer/KeyboardShortcutIntegrationTest.kt:152]: This instantiates a class which wasn't done previously, my intention was a scope function over `player`:

pseudocode:
```kotlin
return mock(AudioPlayer::class.java).apply {
    mReviewer.audioView!!.setPlayer(this)
}
```

Comment (UvrajSB) [reply AnkiDroid/src/test/java/com/ichi2/anki/reviewer/KeyboardShortcutIntegrationTest.kt]: done

**Review by david-allison — APPROVED** [review]
Code looks good! Thanks!

**Review by mikehardy — APPROVED** [review]
Nice!

**Pull request merged**